app/controllers/api/v1/rooms_controller.rb

```diff
@@ -1,15 +1,12 @@
 module Api
   module V1
     class RoomsController < Api::BaseController
+      load_resource :conference, find_by: :short_title
       respond_to :json
 
       def index
-        if params[:conference_id].blank?
-          rooms = Room.all
-        else
-          conference = Conference.find_by_guid(params[:conference_id])
-          rooms = conference.rooms
-        end
+        @conference ? (rooms = @conference.rooms) : (rooms = Room.all)
```

Comment:
It was possible to list all rooms before, so if there is any chance someone used that data to do sth, if we remove it now, things will break.

I think there is small possibility that someone might need that info. The only scenario I can think of is multiple conferences being co-hosted, 1 OSEM instance, but they want to extract overall data for all conferences to do sth with that data.

I'd more focus on what's the harm keeping it?